featuretools/entityset/entity.py

```diff
@@ -414,6 +414,9 @@ def delete_variables(self, variable_ids):
         Remove variables from entity's dataframe and from
         self.variables
         """
+        # check variable is a list
+        if isinstance(variable_ids, str):
+            raise TypeError('strings not accepted, must be of type list')
```

Comment:
Let's change the message to say:

"variable_ids must be a list of variable names"